fdbclient/StorageServerInterface.h

```diff
@@ -72,6 +72,8 @@ struct StorageServerInterface {
 	RequestStream<ReplyPromise<KeyValueStoreType>> getKeyValueStoreType;
 	RequestStream<struct WatchValueRequest> watchValue;
 	RequestStream<struct ReadHotSubRangeRequest> getReadHotRanges;
+	RequestStream<struct SplitRangeRequest> getRangeSplitPoints;
```

Comment:
@etschannen I added two new endpoints in SS interface in recently PRs. Can you help me verify that I handled the serialization correctly?